Cargo.toml

```
@@ -25,17 +25,22 @@ bzip2 = "0.4.3"
 cargo_toml = "0.11.5"
 clap = { version = "3.2.14", features = ["derive"] }
 compact_str = { version = "0.5.2", features = ["serde"] }
+console = "0.15.1"
 crates_io_api = { version = "0.8.0", default-features = false }
+dialoguer = { version = "0.10.2", default-features = false }
 dirs = "4.0.0"
 flate2 = { version = "1.0.24", default-features = false }
 fs4 = "0.6.2"
 futures-util = { version = "0.3.21", default-features = false }
 home = "0.5.3"
+indicatif = "0.16.2"
```

```suggestion
indicatif = "0.17.0"
```

Check [here](https://github.com/console-rs/indicatif/releases/tag/0.17.0) for the release notes, it contains significant performance improvements.

---

It also contains other improvments, like using `once_cell` instead of `lazy_static` and removal of dep `regex`.

---

I just saw that! It wasn't released when I wrote the PR haha